index.html

```diff
@@ -87,18 +91,23 @@ <h2>Alex Pearson</h2>
           </ul>
         </div>
         <div>NINE</div>
-        <div>
-          <div>
-            <h3>Sidney Buckner</h3>
-          </div>
-          <div>
-            <ul>
-              <li>"I make weird faces..."</li>
-              <li><a href="mailto: sidney.buckner2@gmail.com">Email</a></li>
-              <li><a href="https://github.com/SidneyBuckner">GitHub</a></li>
-              <li><a href="https://twitter.com/sidneybuckner18">Twitter</a></li>
-            </ul>
+        <div class="ten">
+          <div id="wallpaper">
           </div>
+            <div>
+              <h2>Hello world, <br>I'm Sidney Buckner</h2>
```

Comment:
This won't block merge, but there's a better way of `br`eaking these two ☝️ lines up (hint: multiple `<h2>` tags!)